src/Root/RootContainer.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router';
+
+import routes from './routes';
+
+
+function Root({ store, history }) {
+  return (
+    <Provider store={store}>
+      <Router history={history} routes={routes} />
+    </Provider>
+  );
+}
+
+Root.propTypes = {
+  history: PropTypes.object.isRequired,
+  store: PropTypes.object.isRequired,
+};
+
+
+export default Root;
```

Comment:
I think this file isn't really a container component. So would it be a good idea if we don't name it `` RootContainer ``?